docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,220 @@
+```bash
+kubectl config use-context dagster
+```
+
+## Step 4: Add the Dagster Helm chart repository
+
+Dagster publishes [Helm charts](https://artifacthub.io/packages/helm/dagster/dagster) for deploying Dagster, with a new chart for each Dagster version.
+
+To install the Dagster Helm charts, run the following command:
+
+```bash
+helm repo add dagster https://dagster-io.github.io/helm
+```
+
+If you have previously added the Dagster Helm charts, run the following command to update the repository:
+
+```bash
+helm repo update
+```
+
+## Step 5: Configure the Helm chart for your deployment
+
+You will need to modify some values in Dagster's Helm chart to deploy your Dagster project.
+
+### Step 5.1: Copy default Helm chart values into values.yaml
+
+Run the following command to copy the values installed from the published Helm charts:
+
+```bash
+helm show values dagster/dagster > values.yaml
+```
+TODO - where to copy this file to?
+
+### Step 5.2: Modify the `values.yaml` file for your deployment
+The `values.yaml` file contains configuration options you can set for your deployment. Different configuration options are explained in inline comments in `values.yaml`, and you can learn more about the different configuration options in [The title of a guide explaining it](/todo).
+
+To deploy your project, you'll need to set the following options:
+- `deployments.name`, which should be a unique name for your deployment
+- `deployments.image.name` and `deployments.image.tag`, which should be set to match the Docker image from Step 2
+- `deployments.dagsterApiGrpcArgs`, which should be set to NEED HELP WITH HOW TO EXPLAIN THIS.
+
+If you are following this guide on your local machine, you will also need to set `pullPolicy: IfNotPresent`. This will use the local version of the image built in Step 2. However, in production use cases when your Docker images are pushed to image registries, this value should remain `pullPolicy: Always`.
+
+<CodeExample filePath="guides/deployment/kubernetes/minimal_values.yaml" language="yaml" title="Minimal changes to make to values.yaml" />
+
+
+
+## Step 6: Install the Helm chart
+Now that you have modified the Helm `values.yaml` file, you can install the changes in your Kubernetes cluster.
+
+Run the following command to install the Helm chart and create a release (TODO i dont really know what create a release means in this context, just took it from the existing guide)
+
+```bash
+helm upgrade --install dagster dagster/dagster -f /path/to/values.yaml
+```
+I think it would be nice to explain the various components of this command and what to set them to, but i dont know what the command is really doing
+
+:::note
+If you are running an older version of Dagster, pass the `--version` flag to `helm upgrade` with the version of Dagster you are running. For example, if you are running `dagster==1.7.4` you'll run the command `helm upgrade --install dagster dagster/dagster -f /path/to/values.yaml --version 1.7.4`
```

Comment:
this is specifically the version of dagster that the system components will use like the webserver and the daemon (you can have your code running on earlier versions)

Reply:
do they need to pass a version flag when copying the `values.yaml` too then?